learning_assistant/management/commands/set_course_prompts.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django management command to generate course prompts.
+"""
+import json
+import logging
+from posixpath import join as urljoin
+
+import requests
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from edx_rest_api_client.auth import SuppliedJwtAuth
+from edx_rest_api_client.client import USER_AGENT
+from opaque_keys.edx.keys import CourseKey
+
+try:
+    from openedx.core.djangoapps.oauth_dispatch.jwt import create_jwt_for_user
+except ImportError:
+    create_jwt_for_user = None
+
+from learning_assistant.models import CoursePrompt
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django Management command to create a set of course prompts
+    """
+
+    def add_arguments(self, parser):
+
+        # list of course ids
+        parser.add_argument(
+            '--course_ids',
+            dest='course_ids',
+            help='comma separated list of course_ids to generate',
+        )
+
+        # pre-message
+        parser.add_argument(
+            '--pre_message',
+            dest='pre_message',
+            help='message to prepend to course topics',
+        )
+
+        parser.add_argument(
+            '--skills_descriptor',
+            dest='skills_descriptor',
+            help='message that describes skill structure'
+        )
+
+        # post-message
+        parser.add_argument(
+            '--post_message',
+            dest='post_message',
+            help='message to append to course topics',
+        )
+
+        # service username
+        parser.add_argument(
+            '-u',
+            '--username',
+            dest='username',
+            help='username for authenticating calls to discovery API'
+        )
+
+    @staticmethod
+    def _get_discovery_api_client(username):
```

Comment:
coverage is down because this function isn't being tested - I'm not really sure it's worth mocking everything in here.